cloudwatch_logs/variables.tf

```diff
@@ -0,0 +1,2 @@
+variable "name" {}
```

Comment:
Maybe a description of this variable would be helpful?

Reply:
What do you mean? Name is perfectly self explanatory :smile: I've put in a description, cheers.